docs/UserGuide.md

```diff
@@ -181,12 +196,13 @@ print “Error: ” and error message for:
 
 ## Command summary
 
-| Action       | Format                                        |
-|--------------|-----------------------------------------------|
-| **Add**      | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS` |
-| **Delete**   | `delete INDEX`                                |
-| **List**     | `list`                                        |
-| **Mark**     | `mark INDEX s/STATUS`                         |
-| **Deadline** | `deadline INDEX d/DEADLINE`                   |
-| **Find**     | `find -FIELD [KEYWORDS]`                      |
-| **Help**     | `help `                                       |
+| Action       | Format                                                                               |
+|--------------|--------------------------------------------------------------------------------------|
+| **Add**      | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS`                                        |
+| **Delete**   | `delete INDEX`                                                                       |
+| **List**     | `list`                                                                               |
+| **Mark**     | `mark INDEX s/STATUS`                                                                |
+| **Deadline** | `deadline INDEX d/DEADLINE`                                                          |
```

Comment:
I think we will have to remove the deadline command since we have transferred the functionality w edit command now